tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -11,7 +11,7 @@ def setUp(self):
         GaiaTestCase.setUp(self)
         curr_time = repr(time.time()).replace('.', '')
 
-        self.contact = MockContact(givenName='Name%s'%curr_time[12:], name='Name%s'%curr_time[12:], familyName='')
+        self.contact = MockContact(givenName='GAIAUITEST_%s'%curr_time[12:], name='GAIAUITEST_%s'%curr_time[12:], familyName='')
```

Comment:
Nit: We can extract the prefix in a variable, so we'll have 1 place to change, the next time we modify it.